packages/cli/src/lib/bundler/types.ts

```diff
@@ -26,6 +26,8 @@ export type BundlingOptions = {
   baseUrl: URL;
   parallelism?: number;
   additionalEntryPoints?: string[];
+  // Path to append to the detected public path, e.g. '/auth'
```

Comment:
One random thought here - if you as an adopter want to segregate the different parts of the site with a reverse proxy, is it harder to make rules matching "anything except /auth", than "specifically /secure needs these preconditions to allow access"? If that makes sense :)

Reply:
Hmm, dunno if that's necessarily true, proxies tend to support ordered/prioritized paths pretty well? Either way this wouldn't be used alongside an authenticating proxy since you'd need to proxy to own the session and therefore the sign-in page.